### PR TITLE
Include svcat commands in the walkthrough

### PR DESCRIPTION
**Pull request by carolynvs**
The [Service Catalog CLI](https://github.com/azure/service-catalog-cli) has hit v1.0 and now supports a set of commands that bring it in line with the functionality of `kubectl`. I'm hoping that we can start mentioning `svcat` in the service catalog documentation so that people are aware that it's an option.

### CODE REVIEW

File: docs/walkthrough.md [hunk 1]
```diff
@@ -6,6 +6,18 @@ If you haven't, please see [install.md](./install.md).
 All commands in this document assume that you're operating out of the root
 of this repository.
 
+Commands are shown below for both `kubectl` and `svcat`. The [Service Catalog CLI](https://github.com/azure/service-catalog-cli),
```
Comment (MHBauer) [inline docs/walkthrough.md]: This belongs as part of the install doc, rather than here.

**Comment by MHBauer** [comment]
The [second commit](https://github.com/kubernetes-incubator/service-catalog/pull/1655/commits/720b9863d74d159ad30efa8692fcb0f6269b800c) is very nice and could be it's own PR.

**Comment by pmorie** [comment]
I talked to @carolynvs and we discussed mentioning this project in one of our md files in this project, but I think this content is better suited at this point for the other's project's site.

I think a lot of the functionality in the svcat project would make a great kubectl plugin, so hopefully we can work together to develop a catalog plugin and drive improvements of the kubectl plugin creation/use experience.

**Comment by carolynvs** [comment]
I've split off the formatting commit into a separate PR in-case you'd like to use it. I'm going to close this for now while I investigate using svcat as a plugin.

**Pull request closed without merging**